content/en/docs/concepts/storage/ephemeral-volumes.md

```diff
@@ -74,8 +74,6 @@ is managed by kubelet, or injecting different data.
 
 ### CSI ephemeral volumes
 
-{{< feature-state for_k8s_version="v1.25" state="stable" >}}
```

Comment:
This fact is still true. Until v1.25 disappears from https://kubernetes.io/docs/home/supported-doc-versions/, we should leave it.

The shortcode tells people when something graduated; we don't link its removal to the removal of the associated feature gate.

Reply:
Thanks for clarifying, in fact, I was a bit torn at the time about whether to delete the state.